app/Containers/AppSection/Authorization/UI/API/Routes/AttachPermissionsToUser.v1.private.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @apiGroup           UserPermission
+ * @apiName            AssignPermissionsToUser
+ *
+ * @api                {GET} /v1/users/permissions/attach Attach Permissions To User
+ * @apiDescription     Attach direct permissions to user
+ *
+ * @apiVersion         1.0.0
+ * @apiPermission      Authenticated ['permissions' => 'manage-permissions', 'roles' => '']
+ *
+ * @apiHeader          {String} accept=application/json
+ * @apiHeader          {String} authorization=Bearer
+ *
+ * @apiBody           {String} user_id
+ * @apiBody           {Array} permission_ids Permission ID or Array of Permissions ID's
+ *
+ * @apiUse            UserSuccessSingleResponse
+ */
+
+use App\Containers\AppSection\Authorization\UI\API\Controllers\AttachPermissionsToUserController;
+use Illuminate\Support\Facades\Route;
+
+Route::post('users/permissions/attach', [AttachPermissionsToUserController::class, 'attachPermissionsToUser'])
```

Comment:
It would be better if you use the format `users/{user_id}/permissions/attach`
So even in the future we can move to policies with no breaking changes!

Reply:
@mderis updated endpoint url's to include user id